packages/components/autocomplete/src/use-autocomplete.tsx

```diff
@@ -60,9 +60,10 @@ type AutocompleteItemWithValue = AutocompleteBaseItem & {
   value?: string
 }
 
-type AutocompleteItemWithItems = AutocompleteBaseItem & {
-  items?: AutocompleteItemWithValue[]
-}
+type AutocompleteItemWithItems = AutocompleteBaseItem &
+  AutocompleteOptionGroupProps & {
```

Comment:
This fix is no longer needed in #2803. Please remove this modification.

Reply:
fixed